src/familybot/lib/family_utils.py

```
@@ -52,120 +53,128 @@ def get_family_game_list_url() -> str:
     return url_family_list
 
 
-def format_message(wishlist: list, short=False) -> str:
+async def format_message(wishlist: list, short=False) -> str:
```

Ruff (0.15.1)

[error] Boolean default positional argument in function definition

(FBT002)